src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -31,7 +30,7 @@ class OgSelection extends DefaultSelection {
    * @return AccountInterface
    */
   public function getAccount() {
-    if (empty($this->account)) {
+    if (empty($this->currentUser)) {
```

Comment:
@damiankloip pointed out we could maybe just mock the current user, instead of having this method. Anyway, not a must for this PR.